### PR TITLE
Replaces uplink management by automated sharing + various misc outfit changes

### PR DESCRIPTION
**Pull request by datlo**
This PR replaces the syndicate uplink management consoles by automatically sharing the bonus TC you get at roundstart/on war declaration to all nuke ops uplinks.

The amount of telecrystals given was not changed.

All maps were changed to fit with the new system.

newnukereadyroom

To make TC sharing more convenient, a stack of 50 TC is now buyable in nuke ops uplink.

The Honk Brand Infiltration Kit now has a regular internal box included, since nukies start with their own one now.

Each uplink now has an uplink_type var that can be set to traitor, nuclear, sit, sst, or admin, depending on what type it is. For now this is not checked by anything but I figured I'd make one for each possible type for the future since I already had to make a nuclear uplink subtype, and it's going to be useful for future PRs I have in mind.

Plasmaman CC officers now start with an appropriate suit instead of the generic grey one. I don't even know if any admins play plasmaman, but hey, now they can.

Changed some of the syndicate outfits to fit with the new uplinks subtypes and also added the syndi box to syndi officer.

🆑
del: Removed syndicate uplink management consoles.
add: Nuclear ops now automatically get bonus TC from round start and war declaration split between their uplinks.
add: Nuke ops can now buy stacks of 50 TCs.
add: Added a regular internals box to the Honk Brand Infiltration Kit.
add: CC officers plasmamen now get an appropriate suit.
tweak: Updated admin-outfitted syndicates
/🆑

### CODE REVIEW

**Comment by Shadeykins** [comment]
amen

**Comment by Denthamos** [comment]
Thank goodness.  No more of the nonsense of people trying to shaft newer ops by stealing TC or trying to keep some instead of doing the donate/redistribute.  Will help speed things along too.

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 1]
```diff
@@ -131,16 +132,36 @@ proc/issyndicate(mob/living/M as mob)
...
 		var/obj/machinery/nuclearbomb/syndicate/the_bomb = new /obj/machinery/nuclearbomb/syndicate(nuke_spawn.loc)
 		the_bomb.r_code = nuke_code
 
 	return ..()
 
+/datum/game_mode/nuclear/proc/scale_telecrystals()
+	var/danger
+	danger = GLOB.player_list.len
+	while(!IsMultiple(++danger,10))//Round up to the nearest multiple of ten
+	total_tc += danger * NUKESCALINGMODIFIER
```
Comment (Birdtalon) [inline code/game/gamemodes/nuclear/nuclear.dm]: Is this supposed to be within your whole loop?

Comment (datlo) [reply code/game/gamemodes/nuclear/nuclear.dm]: No, the loop is one line only and ups the danger var to the next multiple of 10. That was taken from the original uplink management code, I'll make it nicer.

**Pull request merged**